Makefile

```diff
@@ -0,0 +1,45 @@
+DOCKER_COMPOSE = docker-compose
+YARN = $(DOCKER_COMPOSER) exec app yarn
+
+##
+##Project
+##-------
+##
+
+build:
+	$(DOCKER_COMPOSE) up -d --build
+
+kill:
+	$(DOCKER_COMPOSE) stop
+	$(DOCKER_COMPOSE) rm -f
+	$(DOCKER_COMPOSE) down --volumes --remove-orphans
+
+install: ## Install and start the project
```

Comment:
Is the repeated command on the next line really a thing?

Reply:
Take a look at the bottom of the file and run make. You will understand :D.